src/ray/core_worker/core_worker.h

```diff
@@ -20,6 +20,15 @@
 #include "ray/rpc/worker/worker_client.h"
 #include "ray/rpc/worker/worker_server.h"
 
+/// The set of gRPC handlers and their associated level of concurrency. If you want to
+/// add a new call to the worker gRPC server, first change src/ray/protobuf/worker.proto,
+/// then add a new RAY_CORE_WORKER_RPC_HANDLER here, then add the implementation to the
+/// CoreWorker class below.
```

Comment:
Can you be more specific about the naming convention that you need to use for the corresponding code in the protobuf and CoreWorker?

Reply:
Done. Added an example for clarity